MediaBrowser.MediaEncoding/Attachments/AttachmentExtractor.cs

```diff
@@ -164,38 +164,32 @@ await ExtractAttachment(inputFile, protocol, mediaAttachment.Index, outputPath,
                 WindowStyle = ProcessWindowStyle.Hidden,
                 ErrorDialog = false
             };
-            var process = new Process
-            {
-                StartInfo = startInfo
-            };
 
-            _logger.LogInformation("{File} {Arguments}", process.StartInfo.FileName, process.StartInfo.Arguments);
+            int exitCode;
 
-            process.Start();
+            using (var process = new Process { StartInfo = startInfo, EnableRaisingEvents = true })
```

Comment:
Multiple lines

Reply:
Before I go ahead and fix all of these, is this an acceptable way to write this?

```cs
using (var process = new Process
{
    StartInfo = new ProcessStartInfo
    {
        Arguments = string.Format(
            CultureInfo.InvariantCulture,
            "-dump_attachment:{1} {2} -i {0} -t 0 -f null null",
            inputPath,
            attachmentStreamIndex,
            outputPath),
        FileName = _mediaEncoder.EncoderPath,
        UseShellExecute = false,
        CreateNoWindow = true,
        WindowStyle = ProcessWindowStyle.Hidden,
        ErrorDialog = false
    },
    EnableRaisingEvents = true
})
{
    ...
}
```

Reply:
Bump